code/api/src/config/database.json

```diff
@@ -1,22 +1,22 @@
 {
   "development": {
-    "username": "postgres",
+    "username": "travisgm",
```

Comment:
I do wonder about updating this back and forth, but I don't think it harms anything.

Reply:
> I do wonder about updating this back and forth, but I don't think it harms anything.

Yeah I've been wondering about that as well...